lib/cocoapods/generator/xcconfig/xcconfig_helper.rb

```diff
@@ -219,7 +219,7 @@ def self.add_framework_build_settings(framework_path, xcconfig, sandbox_root, in
           build_settings = {
             'FRAMEWORK_SEARCH_PATHS' => quote([dirname]),
           }
-          build_settings['OTHER_LDFLAGS'] = "-framework #{name}" if include_ld_flags
+          build_settings['OTHER_LDFLAGS'] = "-framework \"#{name}\"" if include_ld_flags
```

Comment:
look at line 250 below, does it also need the same change for libraries with spaces?

Reply:
Done